.github/workflows/ci.yml

```diff
@@ -596,6 +600,9 @@ jobs:
 
     - name: Update CMake
       uses: jwlawson/actions-setup-cmake@v1.4
+      with:
+        cmake-version: 3.19.1
+      
```

Comment:
CI is complaining about an extra empty line here.

Reply:
Are we interested in enabling pre-commit.ci? It would automatically fix things like this for us.

Reply:
```suggestion
```